conftest.py

```diff
@@ -90,6 +90,17 @@ def pytest_configure(config: Config):
         env["use_bank"] = False
     if "eth_bank_account" not in env:
         env["eth_bank_account"] = ""
+
+    # Set envs for integration/tests/neon_evm project
+    if "SOLANA_URL" not in os.environ or not os.environ["SOLANA_URL"]:
+        os.environ["SOLANA_URL"] = env["solana_url"]
+    if "EVM_LOADER" not in os.environ or not os.environ["EVM_LOADER"]:
+        os.environ["EVM_LOADER"] = env["evm_loader"]
+    if "NEON_TOKEN_MINT" not in os.environ or not os.environ["NEON_TOKEN_MINT"]:
+        os.environ["NEON_TOKEN_MINT"] = env["spl_neon_mint"]
+    if "CHAIN_ID" not in os.environ or not os.environ["CHAIN_ID"]:
+        os.environ["CHAIN_ID"]: env["network_ids"]["neon"]
+
```

Comment:
maybe it's better to read network settings for neon_evm in neon_evm fixtures ? as in others tests
because it's still a bit confusing 
I can run basic tests as pytest path/to/test --network devnet
but for neon_evm I need to set envs
@lyubenkov @gigimon what do you think

Reply:
@kristinaNikolaevaa it will be better, but this issue just a fastfix (Anton asked). I will create new ticket to rewrite mechanism for neon-evm